Heavy Exploration Rig

comma opened the pull request:
Changes Pathfinder's rig to Heavy Exploration rig
Aside from name and sprite chage:
-No longer locked to pathfinder access
-Glass door of storage renamed to not mention pathfinder
-Melee armor bumped from 35 to 50 (sec default armor vest level)
-Has a bit brighter lamp
-Even worse slowdown/blindness when unpowered.

Also makes helmets actually use brightness values you set on them, and adds a sound to rig pieces sealing (suit storage sound at lower volume)
![](https://i.imgur.com/3dpOt13.png)
![](https://i.imgur.com/PJqwmS4.png)

comma committed:
Actually uses brightness values for helmets

Looks like they were never updated to 0-1 range for lightpower, and were used as just a binary flag.

comma committed:
Added sound to rig piece sealing

Suit storage sound but quieter

AtomicWorm commented:
Default security armor melee resistance is 40, though.

comma commented:
Oh huh, so modular stuff is weaker, I just looked at first vest in armor file, thought it was straight up replacement. Either way IMO it's nothing too crazy for a single suit.

AtomicWorm commented:
Not modular stuff, no. The default security armor has been 40/40/40 (melee/bullet/laser) for years. Modular armor was made reflecting the armor values that were already standard.

I don't know, I feel like 50% melee resistance would've been fine for a hardsuit when they were rare and special. Nowadays everyone seems to have one and that makes it feel a bit OP to me. Or maybe regular armor is just too weak.

I guess the crappy bullet and laser resistance compensates for it well enough. It does make sense that exploration people would have decent melee protection, given what they're most likely to be facing. The regular exploration voidsuit itself could probably use a bit of a melee resistance bump, I suppose.

AtomicWorm commented on maps/torch/items/rigs.dm:
```diff
@@ -295,6 +257,51 @@
+				   /obj/item/weapon/tank,
+				   /obj/item/device/suit_cooling_unit)
+
+	online_slowdown = 0.50
```
I could be wrong, but I'm reasonably sure that worn items have their slowdown values rounded up when the final slowdown calculation is done. I recall @PurpleMartinJCK once telling me that's the reason no apparel items have decimal slowdown values once.

I could be wrong, though.

Spookerton replied:
It's rounded at the end of item slowdown summing and modifying, not per value.

ghostsheet commented on maps/torch/items/rigs.dm:
```diff
@@ -295,6 +257,51 @@
+				   /obj/item/ammo_magazine,
+				   /obj/item/device/flashlight,
+				   /obj/item/weapon/tank,
+				   /obj/item/device/suit_cooling_unit)
```
this is duplicated.
You should also pop in req_access of exploration at least

comma commented:
>I feel like 50% melee resistance would've been fine for a hardsuit when they were rare and special.

There's literally one of this kind, I didn't change rest of hardsuits armor values.

>this is duplicated.

It's not module, for some reason you are allowed to put suit cooler in the suit slot and I don't really see why forbid it if you want TWO suit coolers :V 
Also access locks are poop and I think putting it in access restricted area is enough

ghostsheet commented:
Chinsky that's not what I meant, you've defined allowed twice, scroll up

Roland410 commented:
Exploration access req should be a must if it's not intended for it to be pathfinder only. Not only would it differ from basically every other rig, but it'd make it the easiest rig to get as an antag or otherwise bad guy.

afterthought2 commented:
This is getting out of hand. Move general discussion to feedback on discord.

comma commented:
Bumped armor values around. Bullet 30->20, laser 25->20, energy 35->50 (5 above exp voidsuit)

afterthought2 commented:
rip, dmi conflicts.

comma committed:
Changes Pathfinder's rig to Heavy Exploration rig

Aside from name and sprite chage:
-No longer locked to pathfinder access
-Glass door of storage renamed to not mention pathfinder
-Melee armor bumped from 35 to 50 (sec default armor vest level)
-Has bit brighter lamp
-Even worse slowdown/blindness when unpowered.
PR feedback

Removes duplicate allowed list

Bumped bullet/laser armor down, bumped energy armor a bit up (so it's 5 above the voidsuit)
Resolved conflicts
